docs/builtin_elements.md

```diff
@@ -4,54 +4,53 @@
 
 These properties are valid on all visible items
 
-* **`x`** and **`y`** (*length*): the position of the element relative to its parent
```

Comment:
Is that a mistake?

Reply:
Looks like I accidentally ran a beautifier over that file. I'll back that out again.

When merging the changes will end up in the enum docs anyway.